index.js

```diff
@@ -3,53 +3,168 @@ const state = {
     {
       id: "001-beetroot",
       name: "beetroot",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "002-carrot",
       name: "carrot",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "003-apple",
       name: "apple",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "004-apricot",
       name: "apricot",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "005-avocado",
       name: "avocado",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "006-bananas",
       name: "bananas",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "007-bell-pepper",
       name: "bell pepper",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "008-berry",
       name: "berry",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "009-blueberry",
       name: "blueberry",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
     },
     {
       id: "010-eggplant",
       name: "eggplant",
-      price: 0.35
-    }
+      price: 0.35,
+      quantity: 0,
+    },
   ],
-  cart: []
-};
\ No newline at end of file
+  cart: [],
+};
+
+// rendering store items
+const storeUl = document.querySelector(".store--item-list");
+const cartUl = document.querySelector(".cart--item-list");
+let totalSum = 0;
+let totalNumSpan = document.querySelector(".total-number");
+console.log(totalNumSpan.innerText);
+
+function renderAllStoreItems() {
+  for (let i = 0; i < state.items.length; i++) {
+    let item = state.items[i];
+
+    const storeLi = document.createElement("li");
```

Comment:
function is broken down really nicely and cleanly - easy to follow what is happening